### PR TITLE
Few more s/view/handle/

### PR DESCRIPTION
**Pull request by piotrswigon**
Step by step...

### CODE REVIEW

File: runtime/arc.js [hunk 1]
```diff
@@ -168,20 +168,20 @@ ${this.activeRecipe.toString()}`;
     var arc = new Arc({id: this.generateID(), pecFactory: this._pecFactory, context: this.context, loader: this._loader});
     arc._scheduler = this._scheduler.clone();
     let handleMap = new Map();
-    for (let v of this._views) {
-      let clone = await arc._storageProviderFactory.construct(v.id, v.type, 'in-memory');
-      await clone.cloneFrom(v);
-      handleMap.set(v, clone);
-      if (this._handleDescriptions.has(v)) {
-        arc._handleDescriptions.set(clone, this._handleDescriptions.get(v));
+    for (let h of this._handles) {
```
Comment (dstockwell) [inline runtime/arc.js]: h => handle?

File: runtime/arc.js [hunk 2]
```diff
@@ -168,20 +168,20 @@ ${this.activeRecipe.toString()}`;
...
       }
     };
     this.particleHandleMaps.forEach((value, key) => {
       arc.particleHandleMaps.set(key, {
         spec: value.spec,
-        views: new Map()
+        handles: new Map()
       });
-      value.views.forEach(v => arc.particleHandleMaps.get(key).views.set(v.name, handleMap.get(v)));
+      value.handles.forEach(h => arc.particleHandleMaps.get(key).handles.set(h.name, handleMap.get(h)));
```
Comment (dstockwell) [inline runtime/arc.js]: h => handle?

Comment (piotrswigon) [reply runtime/arc.js]: Done

File: runtime/arc.js (hunk 1 above)
Comment (piotrswigon) [reply runtime/arc.js]: Done

**Pull request merged**